acceptance/bundle/variables/var_file_overrides/output.txt

```diff
@@ -0,0 +1,141 @@
+
+=== variable file
+>>> $CLI bundle validate -o json --var-file=var_files/normal.json
+{
+  "job_cluster_key": "mlops_stacks-cluster",
+  "new_cluster": {
+    "node_type_id": "Standard_DS3_v2",
+    "num_workers": 2
+  }
+}
+
+=== default variable file (see .databricks/*)
+>>> $CLI bundle validate -o json --target with-default-variable-file
+{
+  "job_cluster_key": "mlops_stacks-cluster-2",
+  "new_cluster": {
+    "node_type_id": "Standard_DS3_v3",
+    "num_workers": 9
+  }
+}
+
+=== variable file and variable flag
+>>> $CLI bundle validate -o json --var-file=var_files/normal.json --var=cluster_key=mlops_stacks-cluster-overriden
+{
+  "job_cluster_key": "mlops_stacks-cluster-overriden",
+  "new_cluster": {
+    "node_type_id": "Standard_DS3_v2",
+    "num_workers": 2
+  }
+}
+
+=== variable file and environment variable
+>>> BUNDLE_VAR_cluster_key=incorrectly-overriden $CLI bundle validate -o json --var-file=var_files/normal.json
+{
+  "job_cluster_key": "mlops_stacks-cluster",
+  "new_cluster": {
+    "node_type_id": "Standard_DS3_v2",
+    "num_workers": 2
+  }
+}
+
+=== file not found
+>>> errcode $CLI bundle validate -o json --var-file=var_files/not_found.json
+Error: failed to read variables file: open var_files/not_found.json:<stripped>
+
+
+Exit code: 1
+{
+  "job_cluster_key": "${var.cluster_key}",
+  "new_cluster": {
+    "node_type_id": "${var.cluster.node_type_id}",
+    "num_workers": "${var.cluster_workers}"
+  }
+}
+
+=== file cannot be parsed
+>>> errcode $CLI bundle validate -o json --var-file=var_files/invalid_json.json
+Error: failed to parse variables file: error decoding JSON at :0:0: invalid character 'o' in literal false (expecting 'a')
```

Comment:
> Error: failed to parse variables file:

would be very helpful to print the file name there

`Error: failed to parse variables file var_files/invalid_json.json:`